livre/core/render/Renderer.h

```diff
@@ -28,6 +28,11 @@
 namespace livre
 {
 
+static const uint32_t RENDER_BEGIN = 1;
```

Comment:
To make it explicit that these are bitmask operations I would have defined them as:
RENDER_BEGIN = 1 << 0;
RENDER_FRAME = 1 << 1;
RENDER_END = 1 << 2;
RENDER_ALL = RENDER_BEGIN | RENDER_FRAME | RENDER_END;